app/controllers/api/base_controller/normalizer.rb

```diff
@@ -111,8 +111,8 @@ def normalize_select_attributes(obj, opts)
         end
       end
 
-      def normalize_array(obj)
-        type = @req.subject
+      def normalize_array(obj, type = nil)
+        type ||= @req.subject
```

Comment:
@abellotti I used this for rendering the `miq_groups` properly, and since the `/api` endpoint doesn't have the `groups` subject, I pass in the :groups parameter